angular/src/app/components/main-welcome/main-welcome.component.html

```diff
@@ -5,6 +5,11 @@
                 <img src="assets/Hazmapper-Stack@4x.png" width="250px" />
                 <div style="margin-top: 10px" id="release-version">Version 2.17</div>
             </div>
+            <div id="welcome-banner">
+                Due to a system upgrade on August 7th, 2024, to use HazMapper, please make sure you have logged in at least one time to the  
+                <a href="{{ envService.designSafePortalUrl }}" target="_blank">DesignSafe portal</a> prior to creating maps or adding map
+                data.
```

Comment:
Can we be a bit more explicit with the wording?

Due to a system upgrade on August 7th, 2024, in order to continue using HazMapper users must log in to the <a href="{{ envService.designSafePortalUrl }}" target="_blank">DesignSafe portal</a> at least once prior to creating new maps or adding map data to existing projects.

Reply:
I like it! We need to clear the wording with Tracy as this copy came straight from her.

Reply:
Ohh! I didn't realize that. Don't worry about it if Tracy approves ;)

Reply:
Added these changes